src/components/Movies.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from "react";
+import axios from "axios";
+
+export default class Movies extends Component {
+  constructor() {
+    super();
+    this.state = {
+      movies: [],
+      isError: false,
+      title: "",
+      release: "",
+      description: "",
+    };
+  }
+
+  getMovies = async () => {
+    try {
+      const { data } = await axios.get("https://ghibliapi.herokuapp.com/films");
+      this.setState({
+        movies: data,
+        isError: false,
+      });
+    } catch {
+      this.setState({
+        isError: true,
+      });
+    }
+  };
+
+  componentDidMount() {
+    this.getMovies();
+  }
+
+  handleChange = (e) => {
+    const { value } = e.target;
```

Comment:
As an alternative of destructuring, you can also add a more descriptive variable name i.e.

const userInput = e.target.value;

It makes the code more readable for other developers working on your code. On line 39 it's hard to understand what's being compared: movie.title === value